Controllers/FileReader.cs

```diff
@@ -17,34 +18,38 @@ public void Import()
 
             if (openFileDialog.ShowDialog() == DialogResult.OK)
             {
+                
                 var reader = new StreamReader(openFileDialog.FileName);
                 var line = reader.ReadLine();
-                switch (line.Substring(0, 12))
+                if (line != null && !line.Equals(""))
                 {
-                    case "\"Id\",\"Date\",":
-                        ReadExport(reader, line);
-                        break;
-                    case "\"Sąskaitos N":
-                        ReadSwed(reader);
-                        break;
-                    case "\"SĄSKAITOS  ":
-                    case "SĄSKAITOS  (":
-                    case "ACCOUNT  (LT":
-                        var quotes = false;
-                        if (line[0] == '\"')
-                        {
-                            quotes = true;
-                        }
-
-                        ReadSEB(reader, quotes);
-                        break;
-                    case "Operacijos/B":
-                    case "Transaction ":
-                        ReadLum(reader);
-                        break;
-                    default:
-                        MessageBox.Show("Wrong file type");
-                        break;
+                    switch (line.Substring(0, 12))
```

Comment:
Exception will be raised if line is shorter then 12

Reply:
This isn't the final commit I only wanted to know if the code worked on your computer